packages/x-components/src/x-modules/queries-preview/components/query-preview-list.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <ul>
```

Comment:
I think that we should add classes here, they are not needed right now but just in case we should have classes in `<ul>` and `<li>`